plugins/by-name/octo/default.nix

```
@@ -167,9 +165,16 @@ helpers.neovim-plugin.mkNeovimPlugin {
     };
   };
 
+  extraOptions = {
+    ghPackage = lib.mkPackageOption pkgs "gh" {
+      nullable = true;
+    };
```

I'd be tempted to pass a more verbose name for use in the docs?

```suggestion
    ghPackage = lib.mkPackageOption pkgs "GitHub CLI" {
      default = "gh";
      nullable = true;
    };
```

Can also pass `extraDescription` but I don't think that's useful in this case?